### PR TITLE
Orv2 1966

### PR DESCRIPTION
**Pull request by bcgov-brwang**
# Description

Please provide a summary of the change and the issue fixed. Please include relevant context. List dependency changes.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist

- [ ] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have already been accepted and merged

## Further comments

---

Thanks for the PR!

Deployments, as required, will be available below:
- [Frontend](https://onroutebc-1255-frontend.apps.silver.devops.gov.bc.ca)
- [Vehicles](https://onroutebc-1255-vehicles.apps.silver.devops.gov.bc.ca/api)
- [Dops](https://onroutebc-1255-dops.apps.silver.devops.gov.bc.ca/api)
- [TPS-Migration](https://onroutebc-1255-tps-migration.apps.silver.devops.gov.bc.ca/api)

Please create PRs in draft mode.  Mark as ready to enable:
- [Analysis Workflow](https://github.com/bcgov/onroutebc/actions/workflows/analysis.yml)

After merge, new images are promoted to:
- [Merge Workflow](https://github.com/bcgov/onroutebc/actions/workflows/merge.yml)

### CODE REVIEW

File: vehicles/src/common/enum/orderBy.enum.ts [hunk 1]
```diff
@@ -17,6 +17,7 @@ export enum PermitOrderBy {
   UNIT_NUMBER = 'unitNumber',
   PLATE = 'plate',
   APPLICANT = 'applicant',
+  UPDATED_DATE_TIME = 'updatedDateTime',
```
Comment (praju-aot) [inline vehicles/src/common/enum/orderBy.enum.ts:20]: Already taken up as part of https://github.com/bcgov/onroutebc/pull/1244. Waiting for @zgong-gov FE changes for the PR to be merged. Kindly coordinate with @zgong-gov

File: vehicles/src/common/enum/orderBy.enum.ts [hunk 2]
```diff
@@ -27,4 +28,5 @@ export enum ApplicationOrderBy {
...
```
Comment (praju-aot) [inline vehicles/src/common/enum/orderBy.enum.ts:31]: Already taken up as part of https://github.com/bcgov/onroutebc/pull/1244. Waiting for @zgong-gov FE changes for the PR to be merged. Kindly coordinate with @zgong-gov

File: vehicles/src/modules/permit/application.service.ts [hunk 3]
```diff
@@ -272,6 +272,7 @@ export class ApplicationService {
       unitNumber: 'permitData.unitNumber',
       plate: 'permitData.plate',
       applicant: 'permitData.applicant',
+      updatedDateTime: 'permit.updatedDateTime'
```
Comment (praju-aot) [inline vehicles/src/modules/permit/application.service.ts:275]: Already taken up as part of https://github.com/bcgov/onroutebc/pull/1244. Waiting for @zgong-gov FE changes for the PR to be merged. Kindly coordinate with @zgong-gov

**Review by praju-aot — CHANGES_REQUESTED** [review]

File: frontend/src/features/idir/search/api/idirSearch.ts [hunk 4]
```diff
@@ -47,3 +52,126 @@ export const getCompanyDataBySearch = (
...
+  console.log('test search url: ' + searchURL);
+  searchURL.searchParams.set("searchColumn", searchByFilter);
+  searchURL.searchParams.set("searchString", searchString);
+
+  // API pagination index starts at 1. Hence page + 1.
+  searchURL.searchParams.set("page", (page + 1).toString());
+  searchURL.searchParams.set("take", take.toString());
+  // return httpGETRequest(searchURL.toString()).then((response) => response.data);
+
+  return httpGETRequest(searchURL.toString().replace('application', 'permit').replace('applicationNumber', 'permitNumber'))
```
Comment (praju-aot) [inline frontend/src/features/idir/search/api/idirSearch.ts:75]: Kindly coordinate with @zgong-gov. The response object type is being changed

File: frontend/src/features/idir/search/components/IDIRSearchApplicationLists.tsx [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    ({
+        searchParams,
+      }: {
+        /**
+         * The search parameters entered by the user.
+         */
+        searchParams: SearchFields;
+      }) => {
+
+  const tabs = [
```
Comment (krishnan-aot) [inline frontend/src/features/idir/search/components/IDIRSearchApplicationLists.tsx:17]: Why are you building out a page for applications for staff? A staff user looks at applications in progress inside the company. Can you link the specs your work is based on?

**Review by krishnan-aot — CHANGES_REQUESTED** [review]

**Pull request closed without merging**